src/functionalTest/groovy/com/autonomousapps/jvm/ComputeDominatorTreeSpec.groovy

```diff
@@ -0,0 +1,25 @@
+package com.autonomousapps.jvm
+
+import com.autonomousapps.jvm.projects.ComputeDominatorTreeProject
+
+import static com.autonomousapps.utils.Runner.build
+import static com.google.common.truth.Truth.assertThat
+
+final class ComputeDominatorTreeSpec extends AbstractJvmSpec {
```

Comment:
I know there is an existing [test](https://github.com/autonomousapps/dependency-analysis-gradle-plugin/blob/a22aa2fa499fcb814b2295ee0d6a08efffeebca5/src/functionalTest/groovy/com/autonomousapps/android/DominanceTreeSpec.groovy#L11) for dominator tree, but I couldn't run that one locally, because it depends on some Android config which I didn't have. So I created a new test. Happy to merge them if there are easy instructions for getting the existing test running locally, so that I can update it.

Reply:
Totally fine, I think it makes sense to have a JVM spec for this feature too 👍